app/views/meksis/buildings/index.pdf.erb

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+  <head>
```

Comment:
Bu sayfada `<!DOCTYPE html>, html, head, body` taglarına gerek yok. pdf sayfaları için zaten bir layout var.